airflow/models.py

```diff
@@ -94,9 +94,14 @@ def get_fernet():
     This function could fail either because Cryptography is not installed
     or because the Fernet key is invalid.
     """
-    from cryptography.fernet import Fernet
-    return Fernet(configuration.get('core', 'FERNET_KEY').encode('utf-8'))
-
+    key = configuration.get('core', 'FERNET_KEY')
+    if key != '':
```

Comment:
you know we dont get '', we get u'cryptography_not_found_storing_passwords_in_plain_text'
:(